### PR TITLE
Add in Do Something questions

### PR DESCRIPTION
**Pull request by kevinrobinson**
screen shot 2016-07-15 at 4 48 28 pm

screen shot 2016-07-15 at 4 48 42 pm

### CODE REVIEW

**Comment by kevinrobinson** [comment]
@david-rosales Can you review amigo?

File: ui/src/message_popup/questions.js [hunk 1]
```diff
@@ -297,6 +297,113 @@ export const motivationQuestions:[Question] = [
...
+  },
+  {
+    id: 314,
+    text: 'Students have just returned to class from special and have started the routine of independent reading. You are passing out the snack for the day - bananas. As you reach one student, she asks, "What can we have if we don\'t like bananas?" You say, "Bananas are the only choice. It\'s bananas or you can wait until lunch." The student yells out "WHAT?! I don\'t like bananas," walks out of her seat, rolls up half the rug, and then starts slamming the door repeatedly.',
+    examples: [],
+    nonExamples: []
+  },
+  {
+    id: 315,
+    text: 'The class is lining up for specials in line order. One of your students decides cut in front of another student. You hear the other student ask her to kindly move but she refuses with a smirk on her face. You tell the student she needs to return to her line spot, but she continues to stand there silently. Your student who was cut starts to stomp her feet and yell "That\'s not fair that she gets to cut!"',
```
Comment (david-rosales) [inline ui/src/message_popup/questions.js]: Typo here: "One of your students decides cut in front of another student." => "One of your students decides _**to**_ cut in front of another student."

**Comment by david-rosales** [comment]
@kevinrobinson Looks good, I found one typo but other than that, approved!

File: ui/src/message_popup/questions.js (hunk 1 above)
Comment (kevinrobinson) [inline ui/src/message_popup/questions.js]: awesome, thanks!  fixin'

**Pull request merged**